### ISSUE TITLE
Organizer: Event Invitation results in "Transition Aborted" error

### ISSUE THREAD

**Issue by mariobehling** [fossasia/open-event-frontend#3495]
If a user invites someone as an organizer it shows "Transition Aborted" error.
```
Error: TransitionAborted
    at s (/fastboot/app/assets/vendor-07b444d8eaf097b73d77dd9338ea776f.js:3396:35)
    at y (/fastboot/app/assets/vendor-07b444d8eaf097b73d77dd9338ea776f.js:5237:17)
    at S (/fastboot/app/assets/vendor-07b444d8eaf097b73d77dd9338ea776f.js:5247:12)
    at x (/fastboot/app/assets/vendor-07b444d8eaf097b73d77dd9338ea776f.js:5245:128)
    at E (/fastboot/app/assets/vendor-07b444d8eaf097b73d77dd9338ea776f.js:5243:136)
    at t.invokeWithOnError (/fastboot/app/assets/vendor-07b444d8eaf097b73d77dd9338ea776f.js:4776:242)
    at e.t.flush (/fastboot/app/assets/vendor-07b444d8eaf097b73d77dd9338ea776f.js:4768:74)
    at e.t.flush (/fastboot/app/assets/vendor-07b444d8eaf097b73d77dd9338ea776f.js:4781:358)
    at e.n._end (/fastboot/app/assets/vendor-07b444d8eaf097b73d77dd9338ea776f.js:4843:9)
    at _boundAutorunEnd (/fastboot/app/assets/vendor-07b444d8eaf097b73d77dd9338ea776f.js:4793:574)
    at a (/fastboot/app/assets/vendor-07b444d8eaf097b73d77dd9338ea776f.js:832:61)
    at /fastboot/app/assets/vendor-07b444d8eaf097b73d77dd9338ea776f.js:832:206
    at u (/fastboot/app/assets/vendor-07b444d8eaf097b73d77dd9338ea776f.js:526:5)
    at process._tickCallback (internal/process/next_tick.js:68:7)
```

**Comment by rkpattnaik780** [comment]
Can I work on this issue?

**Comment by iamareebjamal** [comment]
Fixed in #4038 and duplicate of #3347
